content/docs-v1/v1-tutorial.md

```diff
@@ -246,7 +250,8 @@ Once you've played around with adding content and customizing your project, it's
 
 1. In your Terminal, stop the preview process by pressing Control-C.
 2. To create the various formats of your publication type `quire build` and press enter.
-3. In your Finder, navigate to the `X` directory and open the PDF to view it.
+3. Use your Finder or File Explorer to navigate to your project directory.
+4. Navigate to the `public` directory and open the file marked `pdf.html` to view a PDF of your project. 
```

Comment:
`pdf.html` isn't the pdf itself, but rather an HTML file that is used to generate the PDF. I actually don't know where the PDF will ultimately show up